packages/microsoft_defender_endpoint/changelog.yml

```
@@ -1,4 +1,12 @@
 # newer versions go on top
+- version: "3.1.1"
+  changes:
+    - description: Ensure large `$skip` API parameter values are correctly formatted in `vulnerability` data stream.
+      type: bugfix
+      link: https://github.com/elastic/integrations/pull/1
+    - description: 'Enhanced error handling in the CEL program for API calls to prevent "no such key: page_size" errors in the machine data stream.'
+      type: bugfix
+      link: https://github.com/elastic/integrations/pull/1
```

```suggestion
    - description: 'Ensure `page_size` configuration is preserved in CEL evaluation responses.'
      type: bugfix
      link: https://github.com/elastic/integrations/pull/15467
```
This is not really an error handing issue, this is a invariant correctness issue.